test/setup/main.tf

```diff
@@ -183,6 +185,48 @@ module "vpc" {
   }
 }
 
+resource "google_compute_firewall" "allow_private_google_access" {
```

Comment:
If these are needed, can we instead use the `terraform-google-network` module above's `ingress_rules` and `egress_rules` variables?

Reply:
resolved on c637f8a6f8c5e6b70d0df621c2fcd8b7cb93ff47